athenacli/packages/special/iocommands.py

```diff
@@ -219,6 +219,34 @@ def subst_favorite_query_args(query, args):
 
     return [query, None]
 
+@special_command('load', 'load [filename]', 'Load and execute query from a file.')
```

Comment:
Do you mind renaming this to `read` (like the [sqlite3 .read command](https://sqlite.org/cli.html#special_commands_to_sqlite3_dot_commands_))? Just try to use the same name that is used in other DBs.

Reply:
will do